api/app/signals/apps/questionnaires/models/question.py

```diff
@@ -18,7 +18,7 @@ class Question(models.Model):
     Edge models as to how the questions are referred to.
     """
     retrieval_key = models.CharField(unique=True, max_length=255, null=True, blank=True)
-    analysis_key = models.CharField(max_length=255, null=True, blank=True)
+    analysis_key = models.CharField(max_length=255, default='analysis_key_placeholder')
```

Comment:
With this default all questions will have the same analysis_key perhaps it is better to auto populate this (if not given) the same way a slug will be created.